eth2/state_processing/src/per_block_processing/verify_transfer.rs

```diff
@@ -77,19 +95,33 @@ fn verify_transfer_parametric<T: EthSpec>(
         );
     }
 
+    // Load the sender `Validator` record from the state.
     let sender_validator = state
         .validator_registry
         .get(transfer.sender as usize)
         .ok_or_else(|| Error::Invalid(Invalid::FromValidatorUnknown(transfer.sender)))?;
+
     let epoch = state.slot.epoch(spec.slots_per_epoch);
 
+    // Ensure one of the following is met:
+    //
+    // - Time independent checks are being ignored.
```

Comment:
I think this should be _Time dependent checks are being ignored._

Reply:
That said, the "time independence" concept isn't very well named, we could change the variable to something like `allow_future_compatible` (as in, treat as valid any transactions which could become valid in the future)